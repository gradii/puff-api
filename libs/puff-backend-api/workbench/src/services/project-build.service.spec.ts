import * as fs from 'fs';
import git from 'isomorphic-git';


describe('project build service', () => {
  it('test status matrix', async () => {
    const result = await git.statusMatrix({
      fs,
      dir: 'storage/_workbench/projects/qejy2pzvrrkr072hqgs6/code-repo',
    });

    expect(result).toMatchSnapshot();
  });
});
