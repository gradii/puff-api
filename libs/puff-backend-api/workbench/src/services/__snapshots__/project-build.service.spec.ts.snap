// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`project build service test status matrix 1`] = `
Array [
  Array [
    ".editorconfig",
    1,
    1,
    1,
  ],
  Array [
    ".env",
    1,
    1,
    1,
  ],
  Array [
    ".gitignore",
    1,
    1,
    1,
  ],
  Array [
    ".prettierignore",
    1,
    1,
    1,
  ],
  Array [
    ".prettierrc",
    1,
    1,
    1,
  ],
  Array [
    "README.md",
    1,
    1,
    1,
  ],
  Array [
    "angular.json",
    1,
    1,
    1,
  ],
  Array [
    "apps/.gitkeep",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/jest.config.js",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/src/app/.gitkeep",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/src/app/app.controller.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/src/app/app.module.ts",
    1,
    2,
    1,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/actions/common-command-handler.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/actions/init-action.command.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/actions/init-command-handler.service.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/actions/send-another-http-request-action.command.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/actions/send-another-http-request-command-handler.service.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/actions/send-http-request-action.command.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/actions/send-http-request-command-handler.service.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/dddddd-workflow.module.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/events/dddddd.events.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/sagas/dddddd-saga.service.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/dddddd/triggers/default-trigger-cron-trigger.service.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/kill-action-command-handler.service.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/kill-action.command.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/step-status.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/workflow-base.event.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/workflow-saga.service.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/app/workflow/workflow.module.ts",
    0,
    2,
    0,
  ],
  Array [
    "apps/backend/src/assets/.gitkeep",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/src/environments/environment.prod.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/src/environments/environment.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/src/forever-run.js.tpl",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/src/main.ts",
    1,
    2,
    1,
  ],
  Array [
    "apps/backend/tsconfig.app.json",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/tsconfig.json",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/tsconfig.spec.json",
    1,
    1,
    1,
  ],
  Array [
    "apps/backend/tslint.json",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/.browserslistrc",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/jest.config.js",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/proxy.conf.json",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/app/app.component.html",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/app/app.component.scss",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/app/app.component.spec.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/app/app.component.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/app/app.module.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/assets/.gitkeep",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/environments/environment.prod.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/environments/environment.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/favicon.ico",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/index.html",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/main.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/polyfills.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/styles.scss",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/src/test-setup.ts",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/tsconfig.app.json",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/tsconfig.json",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/tsconfig.spec.json",
    1,
    1,
    1,
  ],
  Array [
    "apps/frontend/tslint.json",
    1,
    1,
    1,
  ],
  Array [
    "decorate-angular-cli.js",
    1,
    1,
    1,
  ],
  Array [
    "jest.config.js",
    1,
    1,
    1,
  ],
  Array [
    "libs/.gitkeep",
    1,
    1,
    1,
  ],
  Array [
    "nx.json",
    1,
    1,
    1,
  ],
  Array [
    "package.json",
    1,
    1,
    1,
  ],
  Array [
    "tools/schematics/.gitkeep",
    1,
    1,
    1,
  ],
  Array [
    "tools/tsconfig.tools.json",
    1,
    1,
    1,
  ],
  Array [
    "tsconfig.base.json",
    1,
    1,
    1,
  ],
  Array [
    "tslint.json",
    1,
    1,
    1,
  ],
  Array [
    "yarn.lock",
    1,
    1,
    1,
  ],
]
`;
